azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -174,10 +164,6 @@ public void run() throws Exception {
     generateProperties(propFiles[1]);
   }
 
-  private boolean determineExecuteAsUser(Props sysProps, Props jobProps) {
-    return sysProps.getBoolean(EXECUTE_AS_USER, false);
```

Comment:
@logiclord Thanks for your comment. 

IMHO having option in general is typically a tradeoff between complexity and flexibility. I will comment on the conversation.